src/content/docs/it/guides/backend/turso.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Turso & Astro
+description: Costruisci in locale con un file SQLite e distribuisci globalmente con Turso.
+type: backend
+service: Turso
+stub: false
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Turso](https://turso.tech) è un database basato su libSQL, una fork di SQLite. È ottimizzato per query a bassa latenza, rendendolo adatto ad applicazioni globali.
+
+## Inizializzare Turso in Astro
+
+### Prerequisiti
+
+- La [Turso CLI](https://docs.turso.tech/reference/turso-cli) installata, con accesso eseguito
+- Un Database [Turso](https://turso.tech) con uno schema
+- L'URL del tuo Database
+- Un token di accesso
+
+### Configura le variabili di ambiente
+
+Ottieni l'URL del tuo database usando il comando seguente:
+
+```bash
+turso db show <database-name> --url
+```
+
+Crea un token di autorizzazione per il database:
+
+```bash
+turso db tokens create <database-name>
+```
+
+Aggiungi l'output dei due comandi nel tuo file `.env` nella cartella root del tuo progetto. Se il file non esiste, crealo.
+
+```ini title=".env"
+TURSO_DATABASE_URL=libsql://...
+TURSO_AUTH_TOKEN=
+```
+
+:::caution
+Non usare il prefisso `PUBLIC_` quando crei queste [variabili d'ambiente](/en/guides/environment-variables/) private. Facendolo, esponi questi valori sul client.
```

Comment:
```suggestion
Non usare il prefisso `PUBLIC_` quando crei queste [variabili d'ambiente](/it/guides/environment-variables/) private. Facendolo, esponi questi valori sul client.
```
Just a note that all the links should point to the same language you're translating!